tests/func/test_external_repo.py

```diff
@@ -26,3 +28,13 @@ def test_external_repo(erepo):
             assert path_isin(repo.cache.local.cache_dir, repo.root_dir)
 
         assert mock.call_count == 1
+
+
+def test_external_repo_import_without_remote(erepo, dvc_repo):
```

Comment:
Btw, seems like we don't even need this test, as it is automatically tested by the erepo fixture used in all other tests.